test/test_gpu3.cpp

```diff
@@ -8099,6 +8099,147 @@ TEST_F(NVFuserTest, FusionManagedData_CUDA) {
   ASSERT_EQ(kernel->getManaged<T2>("data2").magic_number, 0x123456789abcdef);
 }
 
+// Repro of issue #2125, 1.45e+03 GB/s on A100-80G
```

Comment:
Is this test supposed to be as fast as 1.4 TB/s? Ran this on A100-80G, and I only see around 130 GB/s. Looks like the inputs are not contiguous, so not sure if this is the exactly the same test as the original repro case.

Reply:
Changed the inputs to contig tensors, but didn't see any perf difference. Can you confirm this test should achieve the bandwidth?

Reply:
This was due to my misunderstanding.